pandas/io/stata.py

```diff
@@ -497,29 +495,10 @@ class ValueLabelTypeMismatch(Warning):
     pass
 
 
-value_label_mismatch_doc = """
-Stata value labels (pandas categories) must be strings. Column {0} contains
-non-string labels which will be converted to strings.  Please check that the
-Stata data file created has not lost information due to duplicate labels.
-"""
-
-
```

Comment:
I'm not a big fan of moving this very long warning to be inline just to make it an f-string.  f-strings should not be dogmatically used when they produce worse code.

Reply:
agreed

Reply:
makes sense, I'll change it in my next commit